### PR TITLE
Testnet resume fixes

### PR DESCRIPTION
**Pull request by da-kami**
Add a few log statements on Bob's side to make the user experience better.
Update / remove ToDos.

I set the log level to `Info` in main again, `Debug` heavily clutters the output. In order to make `Debug` more usable we might want to review printing all those `rpc` messages. But this goes beyond the scope of this PR.

### CODE REVIEW

**Comment by da-kami** [comment]
The last commit prints the Monero confirmations on Bob's side:

```
2021-01-12T05:26:10.458693Z  INFO swap::bitcoin::wallet: Bitcoin tx broadcasted! TXID = 4554251e8fca2910062cf01270f8df51cb455273f87714df3f6f8417f86eb903
2021-01-12T05:26:10.469780Z  INFO swap::protocol::bob::swap: Current state: btc is locked
2021-01-12T05:31:16.427532Z  INFO swap::protocol::bob::swap: Received XMR lock transaction transfer proof from Alice, watching for transfer confirmations
2021-01-12T05:32:12.271309Z  INFO swap::monero::wallet: Monero lock tx received 1 out of 5 confirmations
2021-01-12T05:32:33.954065Z  INFO swap::monero::wallet: Monero lock tx received 2 out of 5 confirmations
2021-01-12T05:35:48.004181Z  INFO swap::monero::wallet: Monero lock tx received 3 out of 5 confirmations
2021-01-12T05:39:44.258748Z  INFO swap::monero::wallet: Monero lock tx received 4 out of 5 confirmations
2021-01-12T05:45:09.890987Z  INFO swap::monero::wallet: Monero lock tx received 5 out of 5 confirmations
2021-01-12T05:45:09.899110Z  INFO swap::protocol::bob::swap: Current state: xmr is locked
2021-01-12T05:45:10.328952Z  INFO swap::protocol::bob::event_loop: Alice acknowledged message 3 received
2021-01-12T05:45:10.335845Z  INFO swap::protocol::bob::swap: Current state: encrypted signature is sent
2021-01-12T05:45:11.486179Z  INFO swap::protocol::bob::swap: Current state: btc is redeemed
2021-01-12T05:45:13.360571Z  INFO swap::protocol::bob::swap: Current state: xmr is redeemed

Process finished with exit code 0
```

I added this for demo purposes but don't see a problem with adding it to production for now, so I added it to this PR :)

File: swap/src/monero/wallet.rs [hunk 1]
```diff
@@ -4,12 +4,15 @@ use async_trait::async_trait;
...
 use monero_harness::rpc::wallet;
 use std::{str::FromStr, time::Duration};
+use tracing::info;
 use url::Url;
 
 use crate::monero::{
     Amount, CreateWalletForOutput, InsufficientFunds, PrivateViewKey, PublicViewKey, Transfer,
     TransferProof, TxHash, WatchForTransfer,
 };
+use bitcoin::hashes::core::sync::atomic::AtomicU32;
```
Comment (D4nte) [inline swap/src/monero/wallet.rs:14]: This should go in the block above.

File: swap/src/monero/wallet.rs [hunk 2]
```diff
@@ -109,33 +112,47 @@ impl WatchForTransfer for Wallet {
...
-                return Err(backoff::Error::Transient(Error::InsufficientConfirmations));
+        let wallet = self.inner.clone();
+
+        let confirmations = Arc::new(AtomicU32::new(0u32));
+        let res = (move || {
+            let confirmations = confirmations.clone();
+            let transfer_proof = transfer_proof.clone();
+            let wallet = wallet.clone();
+            async move {
+                // NOTE: Currently, this is conflating IO errors with the transaction not being
```
Comment (D4nte) [inline swap/src/monero/wallet.rs:123]:
```suggestion
                // NOTE: Currently, this is conflicting IO errors with the transaction not being
```

**Review by D4nte — APPROVED** [review]

**Pull request merged**